frontend/nav.tsx

```diff
@@ -45,6 +56,19 @@ export function Nav({ rep }: { rep: Replicache<M> }) {
             ></path>
           </svg>
         </div>
+        <UndoRedo
+          onClick={() => {
```

Comment:
Has prettier run on this? I'm surprised to see the two `onClick` handlers indent differently since they are same number of chars long.